liquibase-standard/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -1532,19 +1532,6 @@ protected void doMigration() throws Exception {
                 String output = (String) results.getResult("output");
                 outputWriter.write(output);
                 outputWriter.flush();
-                return;
```

Comment:
Shouldn't we keep this in Main for backward compatibility?

Reply:
I am open to keeping it. It is in a protected method, and not in the usual Liquibase class in which we have kept around deprecated methods. For those reasons I thought deleting it would be safe(r). What do you think @filipelautert ?

Reply:
I saw some people calling update method from main in their code. I really doubt someone may be using "SnapshotReference" ... 
So maybe we can drop it and if someone complains about it we can share the new steps.